### ISSUE TITLE
Bug: Repeated refresh of IP without any changes

### ISSUE THREAD

**Issue by techteam-pro-discount** [qdm12/ddns-updater#845]
**TLDR**: *Repeated IP Updates always from same IP to new one, without any changes*

1. Is this urgent: Yes (fills log, risk of ban).
2. DNS provider(s) you use: Cloudflare
3. Program version:

    

    `Running version latest built on 2024-10-20T17:47:49.365Z (commit 949dcd9)`

4. What are you using to run the container: docker Container in Portainer
5. Extra information (optional): 
I have a Fritzbox, last time my IP was changed was on 19.10.2024, still ddns updater reports every 15 minutes that the IP was 104.XX.. and not my IP it should but, but when ever i look in cloudflare or the fritzbox, the IP is correct.
Btw i only use IPV4, not IPV6.

Logs:

```log
Running version latest built on 2024-10-20T17:47:49.365Z (commit 949dcd9)
|       └── all
2024-10-21T07:22:36Z INFO ipv4 address of xyz.de is 104.XX.XX.XXX and your ipv4 address  is 90.XXX.XXX.XX
```

Configuration file (**remove your credentials!**):

```json
{
  "settings": [
    {
      "provider": "cloudflare",
      "zone_identifier": "~snip~",
      "domain": "~snip~",
      "ttl": 1,
      "token": "~snip~",
      "ip_version": "ipv4",
      "ipv6_suffix": ""
    }
  ]
}
```

Host OS:
Ubuntu Server with Portainer

**Comment by techteam-pro-discount** [comment]
For the people who wanna know what i did wrong:
forget so set proxied in the cloudflare settings
